src/site/shared/containers/SelectionPopup/index.tsx

```diff
@@ -78,6 +82,12 @@ export function SelectionPopup({info, modules, docsUrlConfig}: Props) {
         });
     }, input, [setClickThrough]);
 
+    // If placing, deselect all selections and re-render
+    useEffect(() => {
+        if (!isPlacing)
+            history.add(CreateDeselectAllAction(info.selections).execute());
+            renderer.render();
+    }, [isPlacing]);
```

Comment:
I forget if we talked about it -- but I think we decided that deselecting everything was probably not the way to go about this? I believe the SelectionPopup actually has a state variable `clickThrough` that allows mouse events to go straight thru the SelectionPopup, so that's probably what we want? Or we would just want to not show the SelectionPopup while `isPlacing` is true.

Reply:
I probably should have written what we talked about down - I wasn't sure if we had specifically mentioned that, so I included it because I was having trouble with rendering otherwise (the green selection circle would not disappear for some reason). I will definitely take another look with clickThrough now though, because I didn't see that when I was trying to figure out the best way to have the popup disappear.